governator-test-junit/src/main/java/com/netflix/governator/guice/test/AnnotationBasedTestInjectorManager.java

```diff
@@ -234,17 +234,48 @@ protected void configure() {
             bindingBuilder.toInstance(mock);
         }
     }
-    
+
     private static class ArchaiusTestConfigOverrideModule extends AbstractModule {
```

Comment:
Instead of this entire module why not just have a new binding for @RuntimeLayer SettableConfig and override the core set of modules with this override?  Code that's injecting @RuntimeLayer SettableConfig and overriding configuration would just update the RuntimeLayer SettableConfig from the override module